<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Glass3D</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabSteps">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>1920</width>
        <height>100</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="Step1">
       <attribute name="title">
        <string>Paso 1</string>
       </attribute>
       <attribute name="toolTip">
        <string>Selecci칩n de altura del plano de corte</string>
       </attribute>
       <widget class="QLineEdit" name="infoStep1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>461</width>
          <height>24</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Seleccione un punto del molde con clic derecho para generar el centro...</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelStep1Center">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>30</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Centro de la escena:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonNext1">
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>30</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="tabletTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>&amp;Siguiente</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="centerPointText">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>30</y>
          <width>191</width>
          <height>24</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_guide">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>261</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Clic IZQ: Movimiento de la c치mara</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_guide_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>10</y>
          <width>241</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Clic DER: Generar centro</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Step2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Paso 2</string>
       </attribute>
       <attribute name="toolTip">
        <string>Selecci칩n de los vertices del plano</string>
       </attribute>
       <widget class="QPushButton" name="pushButtonBack2">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>30</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>&amp;Volver</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonNext2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>30</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="toolTip">
         <string>Generaci칩n del positivo</string>
        </property>
        <property name="text">
         <string>&amp;Siguiente</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonGenerate2">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>40</y>
          <width>111</width>
          <height>24</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>&amp;Generar Preview</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>241</width>
          <height>21</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="infoStep2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>241</width>
          <height>24</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxWidth">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>10</y>
          <width>62</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>1.500000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>10</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Grosor (mm)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>10</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>170</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>Corte superior</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>40</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>Base interior</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxTop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>10</y>
          <width>62</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxBottom">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>40</y>
          <width>62</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>0</y>
          <width>20</width>
          <height>81</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxDirection">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>10</y>
          <width>111</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Izquierda  (-X)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Derecha   (+X)</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Direcci칩n del escaneo:</string>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>0</y>
          <width>20</width>
          <height>91</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="Line" name="line_3">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>-10</y>
          <width>20</width>
          <height>91</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Step3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Paso 3</string>
       </attribute>
       <widget class="QPushButton" name="pushButtonBack3">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>30</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>&amp;Volver</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSave3">
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>30</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>&amp;Guardar</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxShowMesh">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>Ver &amp;Molde</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxShowPiece">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text">
         <string>Ver &amp;Pieza</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;Ver</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionManual_b_sico"/>
    <addaction name="actionManual_avanzado"/>
    <addaction name="actionInformaci_n"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionReset"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/iconos/abrir.png</normaloff>
     <normalon>:/iconos/abrir.png</normalon>:/iconos/abrir.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Abrir</string>
   </property>
   <property name="toolTip">
    <string>Abrir fichero STL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/iconos/cerrar.png</normaloff>:/iconos/cerrar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cerrar</string>
   </property>
   <property name="toolTip">
    <string>Cerrar Glass3D</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/iconos/reiniciar.png</normaloff>:/iconos/reiniciar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reiniciar</string>
   </property>
   <property name="toolTip">
    <string>Reiniciar vista</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/iconos/preferencias.png</normaloff>:/iconos/preferencias.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferencias</string>
   </property>
   <property name="toolTip">
    <string>Preferencias de Glass3D</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionManual_b_sico">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/iconos/pdf.png</normaloff>:/iconos/pdf.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Manual b치sico</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionManual_avanzado">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/iconos/pdf.png</normaloff>:/iconos/pdf.png</iconset>
   </property>
   <property name="text">
    <string>Manual avanzado</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+M</string>
   </property>
  </action>
  <action name="actionInformaci_n">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/iconos/info.png</normaloff>:/iconos/info.png</iconset>
   </property>
   <property name="text">
    <string>Informaci칩n</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButtonNext1</tabstop>
  <tabstop>comboBoxDirection</tabstop>
  <tabstop>pushButtonGenerate2</tabstop>
  <tabstop>doubleSpinBoxWidth</tabstop>
  <tabstop>doubleSpinBoxTop</tabstop>
  <tabstop>doubleSpinBoxBottom</tabstop>
  <tabstop>pushButtonBack2</tabstop>
  <tabstop>pushButtonNext2</tabstop>
  <tabstop>checkBoxShowMesh</tabstop>
  <tabstop>pushButtonBack3</tabstop>
  <tabstop>pushButtonSave3</tabstop>
 </tabstops>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
